src/biotite/sequence/annotation.py

```diff
@@ -804,7 +805,8 @@ def __setitem__(self, index, item):
             else:
                 seq_start = index.start - self._seqstart
             if index.stop is None:
-                index.start = len(self._sequence)
+                seq_stop = len(self._sequence)
+                index = slice(index.start, seq_stop, index.step)
```

Comment:
I think modifying he index is not neccessary here: If `index.stop` is None, the `Annotation` should include all features after `index.start` which is the desired behavior (provided the fix in my commit is implemented).

```suggestion
```

Reply:
I agree! Committed your change